views/homepage.html

```diff
@@ -0,0 +1 @@
+<h1>HomePage</h1>
```

Comment:
Just because it requires you to have a valid user session and user information in the handler, I would like you to add a message here that's greeting the user by name.